ubuntu/hip/Dockerfile

```diff
@@ -10,7 +10,8 @@ COPY package.py /opt/spack/var/spack/repos/builtin/packages/llvm-amdgpu/package.
 COPY no_cyclades.patch /opt/spack/var/spack/repos/builtin/packages/llvm-amdgpu/no_cyclades.patch
 
 # We need this patch https://github.com/trws/spack/blob/2a54f77d7ebdfbbf131d43abd941ba7736a09737/var/spack/repos/builtin/packages/llvm/package.py#L342-L343
-RUN spack install --deprecated hip@${hip_version}
+RUN spack install --deprecated hip@${hip_version} rocprim@${hip_version}
```

Comment:
Should this be an AND or a "install hip USING" this? I think for the latter you need to start it with `^`

Reply:
It's an AND - rocprim is a separate package (which I thought would already be pulled in by HIP, but is not).

Reply:
Gotcha! Then this looks good ;)